### ISSUE TITLE
Formatted cell Copy-Past blocks Onlyoffice spreadsheet

### ISSUE THREAD

**Issue by stafula** [ONLYOFFICE/onlyoffice-owncloud#187]
On;
- Onlyoffice Spreadsheet editor version 5.1.0 CE 
and;
- Firefox Quantum 60.0.1 (64-bit) @ Updated Windows 10 
- Google Chrome Version 66.0.3359.181 (64-bit  @ Updated Windows 10 
(and more...)

The use case:
The user Copy-Past a cell (with non plain text content) in an Onlyoffice Spreadsheet.

Issue:
after this Copy-Past, no more editing is allowed (on this particular Spreadsheet).
formatted cell Copy-Past crashes the spreadsheet, only a browser refresh can restore the edit capability. 
Onlyoffice auto-save also stops to work and the trigger Copy-Past change is lost.

Exception: 
The problem does not appear when the  Onlyoffice Spreadsheet is opened in more than one browser (shared??)

**Comment by Rita-Bubnova** [comment]
Hello, @stafula.
Can you attach the file to which the problem is reproduced?

**Comment by stafula** [comment]
In my Owncloud/Onlyoffice deploy, one can reproduce this with a blank spreadsheet.
I just need to open a clean spreadsheet, type something in a cell, change the font to bold, select it, copy it and then past it on another cell.
After that, the Autosave stops and we cant edit anything else until a page refresh.

Anyway, here is one of my test spreadsheets;
[test.xlsx](https://github.com/ONLYOFFICE/onlyoffice-owncloud/files/2044989/test.xlsx)

**Comment by Rita-Bubnova** [comment]
Can you send steps to reproduce? How do you perform copy and paste actions: using the context menu or hotkeys? If you use toolbar buttons or menu items it initiates a pop-up window. Try to turn off any addons or plugins that may prevent that. Make a result screenshot with browser console so we able to know what script exactly is stops.

**Comment by stafula** [comment]
yes, of course.
all the above-described copy and paste actions are (system wide) Ctr+c and Ctr+v.
There is no difference if I use Ctr+v or Shift+Ins as Past action.
Note that I'm not reporting a context menu Copy-Past issue.

If you can describe me how to do it, namely what part of the code should be relevant, I can try to extract (and attach here) a JavaScript stack trace...

**Comment by stafula** [comment]
Here are some lines from the chrome java script console;
```
app.js:21 Uncaught TypeError: n.asc_getX is not a function
    at U.i.onShowSpecialPasteOptions (app.js:21)
    at M (app.js:8)
    at n.<anonymous> (app.js:8)
    at n.<anonymous> (app.js:8)
    at n.trigger (sdk-all-min.js:598)
    at n.showSpecialPasteOptions (sdk-all.js:7896)
    at n.trigger (sdk-all-min.js:598)
    at z.Qff (sdk-all.js:8157)
    at n.e$ (sdk-all.js:7929)
    at Ia.Zad (sdk-all-min.js:721)
```

**Comment by ShockwaveNN** [comment]
This is a duplicate of https://github.com/ONLYOFFICE/DocumentServer/issues/270 and was fixed in 5.1.2

**Comment by stafula** [comment]
thank you. 
We will update
